src/index.js

```diff
@@ -1,4 +1,5 @@
 import './style.css';
+/*
 import UI from './modules/UI.js';
 import Store from './modules/localstorage.js';
 import Todostatus from './modules/toDoStatus.js';
```

Comment:
- [OPTIONAL] To enhance the professionalism of your project, it is recommended that you remove any commented-out code. This is important because commented-out code can clutter the codebase, making it harder to read and understand :+1:.